### ISSUE TITLE
Good, but syntax problems in code.

### ISSUE THREAD

**Issue by rickhenderson** [dotnet/docs#27621]
It's great to finally see some well-written tutorials on the .Net site. I'm an oldschool command line programmer so all these new bells and whistles are pretty crazy, but I love how this tutorial starts with VS Code, uses dotnet command line commands, and explains the topics well with a good flow.

Now I can learn all about Unit Tests for .NET.
However: The code for Unit Testing uses String.Format and String.Empty in all cases except for one instance. The String class is not present in this context, and **string.Empty** or **string.Format** has to be used.

Error message from running the code:
```
PS C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects> dotnet test StringLibraryTest/StringLibraryTest.csproj
  Determining projects to restore...
  All projects are up-to-date for restore.
  StringLibrary -> C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibrary\bin\Debug\net6.0\StringLibrary.dll
C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\UnitTest1.cs(18,20): error CS0103: The name 'String' does not exist in the current context [C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\StringLibraryTeC:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\UnitTest1.cs(33,20): error CS0103: The name 'String' does not exist in the current context [C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\StringLibraryTest.csproj]
C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\UnitTest1.cs(42,29): error CS0103: The name 'String' does not exist in the current context [C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\StringLibraryTest.csproj]
C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\UnitTest1.cs(47,20): error CS0103: The name 'String' does not exist in the current context [C:\Users\rihenderson\Documents\Code samples\ClassLibraryProjects\StringLibraryTest\StringLibraryTest.csproj]
```

[Edited to render document details properly.]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a3cbb75b-e0aa-13c6-c07d-65f0f0f68da2
* Version Independent ID: 06ecbd1e-dad7-fbac-9160-dfce34df4d2c
* Content: [Create a .NET class library using Visual Studio Code - .NET](https://docs.microsoft.com/en-us/dotnet/core/tutorials/library-with-visual-studio-code?pivots=dotnet-6-0)
* Content Source: [docs/core/tutorials/library-with-visual-studio-code.md](https://github.com/dotnet/docs/blob/main/docs/core/tutorials/library-with-visual-studio-code.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

**Comment by Youssef1313** [comment]
The article is making use of a new .NET 6 feature, [implicit usings](https://docs.microsoft.com/en-us/dotnet/core/tutorials/top-level-templates#implicit-using-directives), which you need to enable via `<ImplicitUsings>enable</ImplicitUsings>` in a `PropertyGroup` in your csproj file.

However, I think the sample should use `string` anyway since this is the very very common convention and good practice.

**Comment by IEvangelist** [comment]
Closed in #27743
